packages/@react-spectrum/datepicker/stories/DateField.stories.tsx

```diff
@@ -214,9 +214,16 @@ export const IsDateUnavailable: DateFieldStory = {
       return date.compare(new CalendarDate(1980, 1, 1)) >= 0 
           && date.compare(new CalendarDate(1980, 1, 8)) <= 0;
     },
-    errorMessage: 'Date unavailable.'
+    errorMessage: 'Date unavailable.',
+    contextualHelp: (
+      <ContextualHelp>
+        <Heading>Which dates are unavailable?</Heading>
+        <Content>Any date between 1/1/1980 and 1/8/1980 are unavailable.</Content>
+      </ContextualHelp>
+    )
   },
-  name: 'isDateUnavailable'
+  name: 'isDateUnavailable: 1/1/1980 - 1/8/1980 unavailable',
```

Comment:
This ends up being too long. I think the original name with the description and contextual help you added will be enough.